buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/fixtures/TestFixturesPlugin.kt

```diff
@@ -55,9 +55,9 @@ open class TestFixturesPlugin : Plugin<Project> {
         }
 
         configureAsConsumer()
+        removeTestFixturesFromArchivesConfiguration()
```

Comment:
I think we should only remove the testFixtures if we are actually producing them. So I would put that block here: https://github.com/gradle/gradle/blob/6add47450a8453ace39a16a520c9a3b4c8789dce/buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/fixtures/TestFixturesPlugin.kt#L99-L103

Reply:
Thanks. Actually I put it here https://github.com/gradle/gradle/blob/4e56cab882f9b9fad775752bab8d52ee58c7aa66/buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/fixtures/TestFixturesPlugin.kt#L123 because there're two blocks producing them.